configuration/klippy/beacon_true_zero_correction.py

```
@@ -0,0 +1,430 @@
+			logging.info(f"{self.name}: beacon is not configured, beacon true zero correction disabled.")
+		
+	def _handle_connect(self):
+		self.ratos = self.printer.lookup_object('ratos')
+		self.ratos_z_offset = self.printer.lookup_object('ratos_z_offset')
+		self.toolhead = self.printer.lookup_object("toolhead")
+
+		if self.config.has_section("dual_carriage"):
+			self.dual_carriage = self.printer.lookup_object("dual_carriage", None)
+
+		self.orig_cmd = self.gcode.register_command(BEACON_AUTO_CALIBRATE, None)
+		if self.orig_cmd == None:
```

Ruff (0.12.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)